graphitesend/graphitesend.py

```diff
@@ -98,17 +92,15 @@ def clean_metric_name(self, metric_name):
         """
         if not self._clean_metric_name:
             return metric_name
-        metric_name = metric_name.replace('(', '_').replace(')', '')
-        metric_name = metric_name.replace(' ', '_').replace('-', '_')
-        metric_name = metric_name.replace('/', '_').replace('\\', '_')
+        for _from, _to in self.cleaning_replacement_list:
+            metric_name = metric_name.replace(_from, _to)
         return metric_name
 
     '''Format a metric, value, and timestamp for use on the carbon text socket.'''
     def __call__(self, metric_name, metric_value, timestamp=None):
```

Comment:
```
def __call__(self, metric_name, metric_value, timestamp=None):
         if timestamp is None:
              timestamp = time.time()
         timestamp = int(timestamp)
```

I wonder if this would be cleaner by setting the default kwarg as the current time

````
def __call__(self, metric_name, metric_value, timestamp=time.time()):
     timestamp = int(timestamp)
```